.github/workflows/dev.yml

```diff
@@ -0,0 +1,44 @@
+name: DEV
+
+on:
+  push:
+  pull_request:
+jobs:
+  build-deploy:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+    services:
+      jetstream:
+        image: synadia/jsm:latest
+        options: >-
+          -e JSM_MODE=server
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup app dependencies
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Cache Dependencies
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/package.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-npm-
+      - name: Install dependencies
+        run: npm install
+      - name: Check linting
+        run: npm run fix:tslint
```

Comment:
I think in principle it would be possible for `tslint`/`eslint` fixes to change the runtime behaviour of the code and so maybe we don't want to run fix here? Maybe we run `tslint` and fail if there are any warnings/errors

Reply:
btw, I think `eslint` has superceded `tslint` so better to use that in the future, I think

Reply:
true that, I'll do that update in another task.